### PR TITLE
build: internal metrics for packaging.

### LINKED ISSUE #13926: Report how Grafana has been installed
In our anonymous usage stats we should report how Grafana has been installed. We could, through the start scripts make sure Grafana knew if is't running in Docker, has been installed with a debian package and so on.

### PR DESCRIPTION
**Pull request by xlson**
Tracks how Grafana was packaged (docker, rpm, dev and so on)

Closes #13926

### CODE REVIEW

**Review by marefr — COMMENTED** [review]
LGTM

Only thing to discuss is naming. `packaging` argument to main.go is described as "describes the way Grafana was installed" so maybe should be named something related to installation?

**Review by torkelo — APPROVED** [review]
LGTM

**Comment by xlson** [comment]
@marefr not entirely sure what we'd call it. I have a really hard time finding a word to properly describe what it is. Probably because we are using the flag for a few different things at once.

File: packaging/docker/run.sh [hunk 1]
```diff
@@ -80,6 +80,7 @@ fi
 exec grafana-server                                         \
   --homepath="$GF_PATHS_HOME"                               \
   --config="$GF_PATHS_CONFIG"                               \
+  --packaging docker                                        \
```
Comment (torkelo) [inline packaging/docker/run.sh]: should there be an equal sign here?

**Comment by torkelo** [comment]
Yea, hard to come up with a better alternative

* source
* pkg 
* context
* package
* fromPkg
* packaging 

Almost like the packaging (current) or the shorter pkg the most.

File: packaging/docker/run.sh (hunk 1 above)
Comment (xlson) [reply packaging/docker/run.sh]: @torkelo bit works without too, but I'll add it in to make the changes more consistent.

**Review by marefr — APPROVED** [review]

**Pull request merged**